main.go

```diff
@@ -22,7 +22,13 @@ type Params struct {
 	Username      *string  `json:"username,omitempty"`
 }
 
+var (
+	buildDate string
```

Comment:
I'm not fixed on the time thing, you are right with the repeatable builds. But I would keep it currently like it is and update the plugins somehow in a batch.